app/controllers/api/stages_controller.rb

```diff
@@ -36,4 +50,17 @@ def deploy_group_ids
   def deploy_groups
```

Comment:
Fixed. New params structure is: 

{ stage: { source_stage_id: , name: , deploy_group_ids: } }